Dockerfile

```diff
@@ -1,94 +1,115 @@
 FROM ubuntu:14.04
 MAINTAINER support@civisanalytics.com
 
-# Ensure UTF-8 locale.
-RUN locale-gen en_US.UTF-8
+# We need this file to install the right version of numpy.
+COPY requirements.txt /requirements.txt
```

Comment:
The requirements is at the top in order to install numpy right after the build process. This can be moved. I was just trying to get this to work once.